### ISSUE TITLE
can we use an alternative vocoder?

### ISSUE THREAD

**Issue by ttambe** [NVIDIA/flowtron#38]
Quick question: Is it possible to do inference with an alternative vocoder such as wavernn instead of using waveglow?

Thanks,

**Comment by rafaelvalle** [comment]
Yes, it is possible as long as it is trained on the same mel-configuration as Flowtron.
